From f8d1a6ca512940688d2b6e6a212b10dcdd8fe5d9 Mon Sep 17 00:00:00 2001
From: Ziyuan Xu <xzy.xu@rock-chips.com>
Date: Fri, 2 Oct 2020 15:09:12 +0800
Subject: [PATCH] ARM: configs: rv1126-battery.config: enable sound feature

They have a minuscule impact on boot time, codec and simple-card spend a
lot of time as far as we know.

Tested on rv1126-battery-v10 board:
[   17.492609 ] probe of 4-0018 returned 1 after 2116 usecs
[   17.492915 ] initcall es8311_i2c_driver_init+0x0/0x1000 [snd_soc_es8311] returned 0 after 2405 usecs
[   17.510095 ] asoc-simple-card i2s0-sound: ES8311 HiFi <-> ff800000.i2s mapping ok
[   17.512578 ] probe of i2s0-sound returned 1 after 20870 usecs

Signed-off-by: Ziyuan Xu <xzy.xu@rock-chips.com>
Change-Id: Ie25e9dde2cd87616cb4777b4ceeebc1083678d42
---
 arch/arm/configs/rv1126-battery.config | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/arch/arm/configs/rv1126-battery.config b/arch/arm/configs/rv1126-battery.config
index 8e58941e8cd9..ed1c9a34ad3c 100644
--- a/arch/arm/configs/rv1126-battery.config
+++ b/arch/arm/configs/rv1126-battery.config
@@ -4,4 +4,19 @@ CONFIG_POWER_RESET_GPIO=y
 # CONFIG_ROCKCHIP_MPP_VDPU2 is not set
 # CONFIG_ROCKCHIP_MPP_VEPU2 is not set
 CONFIG_RTC_DRV_PCF8563=y
+CONFIG_SND=y
+CONFIG_SND_DMAENGINE_PCM=y
+CONFIG_SND_PCM=y
+CONFIG_SND_SIMPLE_CARD=y
+CONFIG_SND_SIMPLE_CARD_UTILS=y
+CONFIG_SND_SOC=y
+# CONFIG_SND_SOC_DUMMY_CODEC is not set
 CONFIG_SND_SOC_ES8311=m
+CONFIG_SND_SOC_I2C_AND_SPI=y
+CONFIG_SND_SOC_ROCKCHIP=y
+# CONFIG_SND_SOC_ROCKCHIP_I2S is not set
+CONFIG_SND_SOC_ROCKCHIP_I2S_TDM=y
+# CONFIG_SND_SOC_ROCKCHIP_PDM is not set
+CONFIG_SND_TIMER=y
+# CONFIG_SND_USB is not set
+CONFIG_SOUND=y
-- 
2.35.3

